datafusion/core/tests/sql/timestamp.rs

```diff
@@ -968,52 +968,52 @@ async fn timestamp_array_add_interval() -> Result<()> {
     let sql = "SELECT ts, ts - INTERVAL '8' MILLISECONDS FROM table_a";
     let actual = execute_to_batches(&ctx, sql).await;
     let expected = vec![
-        "+----------------------------+-----------------------------------+",
-        "| ts                         | table_a.ts - IntervalDayTime(\"8\") |",
-        "+----------------------------+-----------------------------------+",
-        "| 2020-09-08T13:42:29.190855 | 2020-09-08T13:42:29.182855        |",
-        "| 2020-09-08T12:42:29.190855 | 2020-09-08T12:42:29.182855        |",
-        "| 2020-09-08T11:42:29.190855 | 2020-09-08T11:42:29.182855        |",
-        "+----------------------------+-----------------------------------+",
+        "+----------------------------+----------------------------------------------+",
+        "| ts                         | table_a.ts - IntervalMonthDayNano(\"8000000\") |",
+        "+----------------------------+----------------------------------------------+",
+        "| 2020-09-08T13:42:29.190855 | 2020-09-08T13:42:29.182855                   |",
+        "| 2020-09-08T12:42:29.190855 | 2020-09-08T12:42:29.182855                   |",
+        "| 2020-09-08T11:42:29.190855 | 2020-09-08T11:42:29.182855                   |",
+        "+----------------------------+----------------------------------------------+",
     ];
     assert_batches_eq!(expected, &actual);
 
     let sql = "SELECT ts, ts + INTERVAL '1' SECOND FROM table_b";
     let actual = execute_to_batches(&ctx, sql).await;
     let expected = vec![
-        "+----------------------------+--------------------------------------+",
-        "| ts                         | table_b.ts + IntervalDayTime(\"1000\") |",
-        "+----------------------------+--------------------------------------+",
-        "| 2020-09-08T13:42:29.190855 | 2020-09-08T13:42:30.190855           |",
-        "| 2020-09-08T12:42:29.190855 | 2020-09-08T12:42:30.190855           |",
-        "| 2020-09-08T11:42:29.190855 | 2020-09-08T11:42:30.190855           |",
-        "+----------------------------+--------------------------------------+",
+        "+----------------------------+-------------------------------------------------+",
+        "| ts                         | table_b.ts + IntervalMonthDayNano(\"1000000000\") |",
```

Comment:
This is pretty terrible formatting for the column name 🤮  but I don't think it is fixable easily -- see #2027 for more discussion